--- uvgl-skel-23.03/i3wm/en/etc/skel/.config/i3/config	2023-02-28 01:11:30.000000000 +0900
+++ uvgl-skel/i3wm/en/etc/skel/.config/i3/config	2023-03-01 21:32:55.159551359 +0900
@@ -58,6 +58,9 @@
 # Launch cbatticon
 exec_always --no-startup-id cbatticon
 
+# Launch Volume Icon
+exec_always --no-startup-id volumeicon
+
 # Launch clipman
 exec_always --no-startup-id xfce4-clipman
 
@@ -79,9 +82,6 @@
 # Launch Fcitx5
 exec_always --no-startup-id fcitx5
 
-# Launch volumeicon
-exec_always --no-startup-id volumeicon
-
 # Open shortcuts list
 bindsym $mod+c exec --no-startup-id xdg-open ~/.config/i3/shortcuts.list
 
