--- anthy-unicode/mkworddic/Makefile.am	2021-12-24 21:44:30.000000000 +0900
+++ anthy-unicode-patched/mkworddic/Makefile.am	2023-07-18 22:26:08.534641476 +0900
@@ -2,7 +2,7 @@
 
 # Files
 noinst_SCRIPTS =
-EXTRA_DICS = base.t extra.t compound.t name.t adjust.t utf8.t tankanji.t
+EXTRA_DICS = base.t extra.t compound.t name.t adjust.t utf8.t tankanji.t neologd.t place-names.t sudachidict.t
 ZIPCODE_DICT = zipcode.t
 CANNADIC_DIST = gcanna.ctd gcannaf.ctd
 INCLUDES = -I$(top_srcdir)/
@@ -23,7 +23,10 @@
 	    @top_srcdir@/mkworddic/compound.t \
 	    @top_srcdir@/mkworddic/extra.t \
 	    @top_srcdir@/alt-cannadic/g_fname.t \
-	    @top_srcdir@/mkworddic/udict
+	    @top_srcdir@/mkworddic/udict \
+	    @top_srcdir@/anthydic-nk/neologd.t \
+	    @top_srcdir@/anthydic-nk/place-names.t \
+	    @top_srcdir@/anthydic-nk/sudachidict.t
 
 # Generate the dictionary
 noinst_PROGRAMS = mkworddic
