--- kasumi-unicode-2.6/KasumiDic.hxx	2021-07-14 14:52:57.000000000 +0900
+++ kasumi-unicode-2.6-patch/KasumiDic.hxx	2023-01-27 21:39:05.843547732 +0900
@@ -49,12 +49,12 @@
   list<KasumiWord*> mWordList;
   vector<KasumiDicEventListener*> EventListeners;
   
-  void load(KasumiConfiguration *conf) throw (KasumiException);
+  void load(KasumiConfiguration *conf) noexcept(false);
 public:
     KasumiDic(KasumiConfiguration *conf)
-	throw(KasumiException);
+	noexcept(false);
     ~KasumiDic();
-    void store() throw(KasumiException);
+    void store() noexcept(false);
     void appendWord(KasumiWord *word); // returns this word's ID
     void removeWord(unsigned int id);
 
@@ -69,7 +69,7 @@
     void changedSound(KasumiWord *word);
     void changedWordType(KasumiWord *word);
 
-//  KasumiWord *getWordWithID(unsigned int id) throw(KasumiException);
+//  KasumiWord *getWordWithID(unsigned int id) noexcept(false);
 //  int getUpperBoundOfWordID();
 };
 
