--- kasumi-unicode-2.6/KasumiConfiguration.cxx	2021-07-14 14:52:57.000000000 +0900
+++ kasumi-unicode-2.6-patch/KasumiConfiguration.cxx	2023-01-27 21:50:10.608766310 +0900
@@ -58,7 +58,7 @@
 //     to loadConfigurationFromArgument method
 
 KasumiConfiguration::KasumiConfiguration(int argc, char *argv[])
-  throw(KasumiException){
+  noexcept(false){
 
   try{  
     loadDefaultProperties();
@@ -87,7 +87,7 @@
   saveConfiguration();
 }
 
-void KasumiConfiguration::loadDefaultProperties() throw(KasumiException){
+void KasumiConfiguration::loadDefaultProperties() noexcept(false){
   char *home = getenv("HOME");
   if(home == NULL){
       throw KasumiException(string("Cannot find $HOME environment variable."),
@@ -124,7 +124,7 @@
 }
 
 void KasumiConfiguration::loadConfigurationFromArgument(int argc, char *argv[])
-  throw(KasumiException){
+  noexcept(false){
   int option_index = 0;
   static struct option long_options[] = {
     {"help", no_argument, NULL, 'h'},
@@ -215,7 +215,7 @@
 
 
 void KasumiConfiguration::loadConfigurationFile()
-  throw(KasumiException){
+  noexcept(false){
 
   int line = 0;
   string Contents = string();
@@ -248,12 +248,12 @@
 
 // ToDo: implement saveConfiguration method
 void KasumiConfiguration::saveConfiguration()
-  throw(KasumiException){
+  noexcept(false){
 
 }
 
 void KasumiConfiguration::checkValidity()
-  throw(KasumiException){
+  noexcept(false){
   
   if(config[string("StartupMode")] != string("MANAGE") &&
      config[string("StartupMode")] != string("ADD") &&
