--- kasumi-unicode-2.6/KasumiWord.hxx	2021-07-14 14:52:57.000000000 +0900
+++ kasumi-unicode-2.6-patch/KasumiWord.hxx	2023-01-27 21:39:05.846881132 +0900
@@ -70,9 +70,9 @@
 
     // property functions
     void setSound(const string &aSound)
-      throw(KasumiException);
+      noexcept(false);
     void setSoundByUTF8(const string &aSound)
-      throw(KasumiException);
+      noexcept(false);
     string getSound(){ return Sound; };
     string getSoundByUTF8(){ return Sound_UTF8; };
 
