--- kasumi-unicode-2.6/KasumiWord.cxx	2021-07-14 14:52:57.000000000 +0900
+++ kasumi-unicode-2.6-patch/KasumiWord.cxx	2023-01-27 21:48:04.273315847 +0900
@@ -178,7 +178,7 @@
 }
 
 void KasumiWord::setSound(const string &aSound)
-    throw(KasumiException){
+    noexcept(false){
 
     if(aSound == Sound)
 	return;
@@ -202,7 +202,7 @@
 }
 
 void KasumiWord::setSoundByUTF8(const string &aSound)
-    throw(KasumiException){
+    noexcept(false){
 
     if(aSound == Sound_UTF8)
 	return;
